src/owl-core-lib/elements-owl-core.xsl

```diff
@@ -2,16 +2,18 @@
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
     xmlns:xs="http://www.w3.org/2001/XMLSchema"
     xmlns:math="http://www.w3.org/2005/xpath-functions/math"
-    xmlns:xd="http://www.oxygenxml.com/ns/doc/xsl" xmlns:fn="http://www.w3.org/2005/xpath-functions"
-    exclude-result-prefixes="xs math xd xsl uml xmi umldi dc fn f functx"
+    xmlns:xd="http://www.oxygenxml.com/ns/doc/xsl" 
```

Comment:
yes, like this, we can have a complete set of namespaces here, and mark the ones which shall be excluded from the output